firmware/spade/src/rpi/main.c

```diff
@@ -213,6 +234,9 @@ int main() {
   jerry_init(JERRY_INIT_MEM_STATS);
   init(sprite_free_jerry_object); // TODO: document
 
+  // Initialize SD card
+  mount_sd_card();
```

Comment:
might it make sense to add a debug log, stating if the sd card got mounted or not?